packages/pigeon/lib/linux_generator.dart

```diff
@@ -1134,6 +1099,14 @@ String _snakeCaseFromCamelCase(String camelCase) {
       (Match m) => '${m.start == 0 ? '' : '_'}${m[0]!.toLowerCase()}');
 }
 
+// Converts a string from snake_case to CamelCase
+String _camelCaseFromSnakeCase(String snakeCase) {
+  return snakeCase
+      .split('_')
+      .map((String v) => v[0].toUpperCase() + v.substring(1))
+      .join();
+}
```

Comment:
This should probably go in the generator_tools file, though it seems like there are other functions like this one in language specific files also that should be moved. (I'll do that later)